tests/wpt/web-platform-tests/domparsing/XMLSerializer-serializeToString.html

```diff
@@ -31,6 +31,11 @@ <h1>domparsing_XMLSerializer_serializeToString</h1>
   assert_equals(serialize(root), '<root><child1>value1</child1></root>');
 }, 'check XMLSerializer.serializeToString method could parsing xmldoc to string');
 
+test(function() {
+  assert_equals(typeof(serialize(document)), "string");
```

Comment:
We should check the actual result of serialization here. To make this simple, rather than serializing `document`, let's use DOMParser to parse an HTML document and append a couple simple nodes to the result, then serialize that and check the expected result.

Additionally, we will want a test that verifies that XML documents can be serialized as well. However, this will run into https://github.com/w3c/DOM-Parsing/issues/50, so let's compare against the expected specification output for the time being.

Reply:
@jdm I just try to follow this steps that’s why I appended the nodes in the code